packages/tasks/src/model-libraries.ts

```diff
@@ -586,6 +586,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"cvlm_llama2_tokenizer/tokenizer.model"`,
 		snippets: snippets.seed_story,
 	},
+	soloaudio: {
+		prettyLabel: "SoloAudio",
+		repoName: "SoloAudio",
+		repoUrl: "https://github.com/WangHelin1997/SoloAudio",
+		docsUrl: "https://github.com/WangHelin1997/SoloAudio",
+		snippets: snippets.soloaudio,
```

Comment:
There is no snippets on this PR (maybe forgot to commit?).

Code snippets are interesting when the library is capable to seamlessly download the weights for the user (which we highly recommend!). This makes the workflow easier for new users. You can check [this guide](https://huggingface.co/docs/huggingface_hub/guides/download) on how to do that.

Reply:
Removed that. thanks